### PR TITLE
Fix alignment tool bazel build and update alignment code to python 3

### LINKED ISSUE #19: Bazel build error when trying to run align_fine
I am trying to get the alignment tool working, but am getting the following error:

`/data/master-thesis/src/001_dataset/alignment/BUILD.bazel:52:10: error loading package '@com_google_protobuf//': Label '@bazel_skylib//rules:common_settings.bzl' is invalid because 'rules' is not a package; perhaps you meant to put the colon here: '@bazel_skylib//:rules/common_settings.bzl'? and referenced by '//:align'`

Could you please provide me some help?

Thank you in advance, you are doing a really great job with magenta and note-seq!

**System Information:**
- align_fine code copied from Release v0.0.1
- OS: Ubuntu 16.04
- Python 3.7 using Anaconda w. magenta and note-seq package installed through pip
- Bazel 3.5.0 installed using apt like described [here](https://docs.bazel.build/versions/3.5.0/install-ubuntu.html)

**Full Output:**
```
(ma) sebi@sebi-desktop:/data/master-thesis/src/001_dataset/alignment$ INPUT_DIR=../test/
(ma) sebi@sebi-desktop:/data/master-thesis/src/001_dataset/alignment$ OUTPUT_DIR=../test_fine_align/
(ma) sebi@sebi-desktop:/data/master-thesis/src/001_dataset/alignment$ bazel run :align_fine -- --input_dir "${INPUT_DIR}" --output_dir "${OUTPUT_DIR}"
Starting local Bazel server and connecting to it...
INFO: SHA256 (https://github.com/protocolbuffers/protobuf/archive/master.zip) = 0dd2b6f666dbe89f7e431655b7ed6b26a5e04198b72c985b725f552a0eb5f1e4
DEBUG: Rule 'com_google_protobuf' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "0dd2b6f666dbe89f7e431655b7ed6b26a5e04198b72c985b725f552a0eb5f1e4"
DEBUG: Repository com_google_protobuf instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /home/sebi/.cache/bazel/_bazel_sebi/20d58b7978468f7d4694d017731bad27/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
INFO: Repository eigen_repo instantiated at:
  no stack (--record_rule_instantiation_callstack not enabled)
Repository rule http_archive defined at:
  /home/sebi/.cache/bazel/_bazel_sebi/20d58b7978468f7d4694d017731bad27/external/bazel_tools/tools/build_defs/repo/http.bzl:336:31: in <toplevel>
ERROR: /data/master-thesis/src/001_dataset/alignment/BUILD.bazel:52:10: error loading package '@com_google_protobuf//': Label '@bazel_skylib//rules:common_settings.bzl' is invalid because 'rules' is not a package; perhaps you meant to put the colon here: '@bazel_skylib//:rules/common_settings.bzl'? and referenced by '//:align'
ERROR: Analysis of target '//:align_fine' failed; build aborted: Analysis failed
INFO: Elapsed time: 8.804s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (19 packages loaded, 76 targets configured)
FAILED: Build did NOT complete successfully (19 packages loaded, 76 targets configured)
```

### PR DESCRIPTION
**Pull request by DerWaldi**
Fixes #19

### CODE REVIEW

**Comment by DerWaldi** [comment]
@googlebot I signed it!

**Comment by cghawthorne** [comment]
Thanks for the fixes! Can you reformat the code to match our 2-space indentation standard? There's a pylintrc here to check it: https://github.com/magenta/note-seq/blob/master/pylintrc

**Comment by DerWaldi** [comment]
You're welcome! I reformatted the could so it should now match the standard.

File: note_seq/alignment/align_fine_lib.py [hunk 1]
```diff
@@ -129,14 +129,14 @@ def align_cpp(samples,
...
-  with open(temp_path, 'w') as f:
+  with open(temp_path, 'wb') as f:
     f.write(alignment_task.SerializeToString())
 
   # Align with C++ program.
   subprocess.check_call([ALIGN_BINARY, temp_path])
 
   # Read file.
-  with open(temp_path + '.result') as f:
+  with open(temp_path + '.result', "rb") as f:
```
Comment (cghawthorne) [inline note_seq/alignment/align_fine_lib.py:139]: we typically use single quotes. can you switch to be consistent?

Comment (DerWaldi) [reply note_seq/alignment/align_fine_lib.py]: Here you are :)

**Review by cghawthorne — APPROVED** [review]

**Pull request merged**